assembl/static2/js/app/components/administration/survey/configureThematicForm.jsx

```diff
@@ -181,6 +182,23 @@ class ConfigureThematicForm extends React.PureComponent<Props> {
         {theme && theme.messageViewOverride && theme.messageViewOverride.value === MESSAGE_VIEW.messageColumns ? (
           <MultiColumnsFields editLocale={editLocale} fieldPrefix={name} nbColumnsInForm={nbColumnsInForm} />
         ) : null}
+        <React.Fragment>
+          <div className="margin-l" />
+          <Helper
+            label={I18n.t('administration.summary')}
+            helperUrl="/static2/img/helpers/helper_BM_1.png"
```

Comment:
Please mark a TODO comment to migrate these to an s3 bucket (if they aren't already). Best to serve these assets from S3 if there's internet (and cache it), because it's faster and frees up resources for the compute instances to resolve other client requests.

Reply:
it is not the only place where `/static2/img/` is used, there is about 35 occurrences of this path scattered in the project. Fonts should also be moved but should be group before. I created a ticket in Jira to migrate those asset to S3 if needed. (https://bluenove.atlassian.net/browse/DEVAS-2054)